### DISCUSSION TITLE
Log files for deb install.

### DISCUSSION THREAD

**Discussion by quldude** [inbucket/inbucket#544] [Q&A] [answered]
Where can i find log files that show errors for an install that uses linux_amd64.deb on ubuntu? Log level is set to error.

**Comment by jhillyerd** [comment, marked as answer]
Logs should be streamed into the systemd journal on debian and redhat.  I've got the start of some [docs on it here](https://book.inbucket.org/installation/linux-package.html#enable--start-inbucket), but the tl;dr is run `journalctl -u inbucket`
